src/deion/worker/views/standings.js

```diff
@@ -65,6 +66,48 @@ async function updateStandings(
                         );
                     }
                     confTeams[j].highlight = confTeams[j].tid === g.userTid;
+
```

Comment:
The code in standings.js and leagueDashboard.js looks repetitive. Maybe it should be put in a function in helpers.js?

Reply:
Yeah, fair point. I'll take a look at implementing that tonight